### ISSUE TITLE
[Feature Request] Add thumbnails in playlist menu.

### ISSUE THREAD

**Issue by RexAckermann** [tomasklaen/uosc#479]
I once came accros a script that shows thumbnail menu in playlist ,that broke.I like uosc's playlist too.I would like see the thumbnail feature on uosc's playlist.
For referance,
[mpv-gallery-view](https://github.com/occivink/mpv-gallery-view)

**Comment by tomasklaen** [comment]
Not planned for now or foreseeable future. Too much work and hacks for a feature I don't personally care about, and would probably disable if it was in.

Also I don't think this would even be possible, since you can't clip images, so they would be poking out of menu boundaries.

**Comment by po5** [comment]
You *can* crop images, by skipping parts of the BGRA. See YouTube thumbnail cutting in marzzzello's thumbnail script.

**Comment by christoph-heinrich** [comment]
Rows can be cropped, but not columns afaik.

**Comment by po5** [comment]
BGRA is raw image data, you can easily omit columns and alter pixels.

**Comment by christoph-heinrich** [comment]
Yeah sure you can modify the image file itself, but that's not necessary for rows.
